forum/style/template/overall_footer.html

```diff
@@ -127,31 +127,31 @@
 
 		<script type="text/javascript">
 			$( document ).ready(function($){
-				<!-- IF S_MERGE_VIEW -->
```

Comment:
These aren't normal HTML comments, but rather phpBB template tags. Converting them to JavaScript comments will break the template. You'll need to revert all changes to this file.